synapse/cores/common.py

```diff
@@ -3749,3 +3759,78 @@ def delBlobValu(self, key):
             NoSuchName: If the key is not present in the store.
         '''
         return self.store.delBlobValu(key)
+
+    def _onSetMembranes(self, membranes):
+        [self._initMembrane(name, rules) for name, rules in membranes]
```

Comment:
What happens if I have one membrane, and then call setConfOpt and pass a the config for the current membrane and then the config for a second membrane?  Wouldn't the end result be based on the order of the config data, since you could get a MembraneExists exception thrown and you'd break out of the list comprehension?
What if a membrane exists and is having updated rules set on it?

Reply:
Ya - that is a good point. The behavior should probably be that if a Membrane already exists, that we update its rules. We just don't want to add another FIFO

Reply:
fixed